sal_interfaces/WeatherForecast/WeatherForecast_Telemetry.xml

```diff
@@ -0,0 +1,674 @@
+      <Count>14</Count>
+      <Description>Maximum skin temperature.</Description>
+      <Units>deg_C</Units>
+      <EFDB_Name>skinTemperatureMax</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>14</Count>
+      <Description>Minimum skin temperature.</Description>
+      <Units>deg_C</Units>
+      <EFDB_Name>skinTemperatureMin</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>14</Count>
+      <Description>Mean skin temperature.</Description>
+      <Units>deg_C</Units>
+      <EFDB_Name>skinTemperatureMean</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>14</Count>
+      <Description>Maximum dew point temperature.</Description>
+      <Units>deg_C</Units>
+      <EFDB_Name>dewPointTemperatureMax</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>14</Count>
+      <Description>Dew point temperature.</Description>
+      <Units>deg_C</Units>
+      <EFDB_Name>dewPointTemperatureMin</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>14</Count>
+      <Description>Mean dew point temperature.</Description>
+      <Units>deg_C</Units>
+      <EFDB_Name>dewPointTemperatureMean</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>long</IDL_Type>
+      <Count>14</Count>
+      <Description>Maximum CAPE.</Description>
+      <Units>unitless</Units>
+      <EFDB_Name>capeMax</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>long</IDL_Type>
+      <Count>14</Count>
+      <Description>Minimum CAPE.</Description>
+      <Units>unitless</Units>
+      <EFDB_Name>capeMin</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>long</IDL_Type>
+      <Count>14</Count>
+      <Description>Mean CAPE.</Description>
+      <Units>unitless</Units>
+      <EFDB_Name>capeMean</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>14</Count>
+      <Description>Maximum lifted index.</Description>
+      <Units>unitless</Units>
+      <EFDB_Name>liftedIndexMax</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>14</Count>
+      <Description>Minimum lifted index.</Description>
+      <Units>unitless</Units>
+      <EFDB_Name>liftedIndexMin</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>14</Count>
+      <Description>Mean lifted index.</Description>
+      <Units>unitless</Units>
+      <EFDB_Name>liftedIndexMean</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>14</Count>
+      <Description>Evapotranspiration.</Description>
+      <Units>millimeter</Units>
+      <EFDB_Name>evapoTranspiration</EFDB_Name>
+    </item>
+    <item>
+      <IDL_Type>float</IDL_Type>
+      <Count>14</Count>
+      <Description>Reference evapotranspiration as defined by FAO(Food and agriculture UN organization) using Penman-Monteith equation.</Description>
```

Comment:
Copy this description to the other `referenceEvapoTranspirationFao` and add a space before `(` in both descriptions.